### ISSUE TITLE
3.5 (61) Choice of example

### ISSUE THREAD

**Issue by Jeff-Cousins** [pat-rogers/Ada-202x-WG9-Informal-Review#29]
"Red .. Green" is perhaps not the best example from 3.5.1 as it involves overloading.

**Comment by ARG-Editor** [comment]
I disagree, this subexample (implicitly) shows that one can use overloaded literals as expected without complication. It is original Ada 95 text as well, so I don't see any good reason to change it.
